Doc/library/dis.rst

```diff
@@ -1676,9 +1700,13 @@ iterations of the loop.
 
 .. opcode:: CALL_INTRINSIC_2
 
-   Calls an intrinsic function with two arguments. Passes ``STACK[-2]``, ``STACK[-1]`` as the
-   arguments and sets ``STACK[-1]`` to the result. Used to implement functionality that is
-   necessary but not performance critical.
+   Calls an intrinsic function with two arguments. Used to implement functionality
+   that is necessary but not performance critical::
```

Comment:
```suggestion
   that is not performance critical::
```

Reply:
I can see you copied this from the 1-arg opcode. But I think it's redundant in both cases.

Reply:
I will update it